src/rootPages/Designer/properties/process/ABProcessTaskServiceInsertRecord.js

```diff
@@ -0,0 +1,470 @@
+            };
+         });
+
+         // field options to the form
+         object.fields().forEach((f) => {
+            $fieldValues.addView({
+               fieldId: f.id,
+               view: "layout",
+               cols: [
+                  {
+                     rows: [
+                        {
+                           view: "label",
+                           label: f.label,
+                           width: 100,
+                        },
+                        { fillspace: true },
+                     ],
+                  },
+                  {
+                     rows: [
+                        {
+                           name: "setSelector",
+                           view: "select",
+                           options: setOptions,
+                           on: {
+                              onChange: function (newVal) {
+                                 const $parent = this.getParentView();
+                                 const $valuePanel = $parent.queryView({
+                                    name: "valuePanel",
+                                 });
+                                 $valuePanel.showBatch(newVal);
+                              },
+                           },
+                        },
+                        {
+                           name: "valuePanel",
+                           view: "multiview",
+                           visibleBatch: 0,
+                           cols: [
+                              { batch: 0, fillspace: true },
+                              { batch: 1, view: "text" },
+                              {
+                                 batch: 2,
+                                 view: "select",
+                                 options: startElemObjFields,
+                              },
+                              {
+                                 batch: 3,
+                                 view: "select",
+                                 options: prevElemObjFields,
+                              },
+                              { batch: 4, view: "text" },
+                              {
+                                 batch: 5,
+                                 view: "select",
+                                 options: repeatObjectFields,
+                              },
+                              {
+                                 batch: 6,
+                                 view: "multicombo",
+                                 label: "",
+                                 options: queryTaskOptions,
+                              },
+                           ],
+                        },
+                     ],
+                  },
+               ],
+            });
+         });
+      }
+
+      /**
+       * @method propertiesStash()
+       * pull our values from our property panel.
+       * @param {string} id
+       *        the webix $$(id) of the properties panel area.
+       */
+      propertiesStash() {
```

Comment:
our new Property views use `populate()`  and `values()`  now.  refactor this old `propertiesStash()` to be `values()`

Reply:
or just get rid of this if it is unnecessary